internal/server/stat/golden/get_stat_all_test.go

```diff
@@ -80,4 +77,9 @@ func TestGetStatAll(t *testing.T) {
 			}
 		}
 	}
+
+	if err := e2e.TestWithImportGroupLatency(
+		"GetStatAll", &e2e.TestOption{}, testSuite); err != nil {
+		t.Errorf("TestWithImportGroupLatency() = %s", err)
+	}
```

Comment:
neat!

Reply:
latencyTest() and goldenTest() are called in e2e.TestWithImportGroupLatency().

but I see your confusion -- renamed TestWithImportGroupLatency to TestDriver().